openfisca_us/variables/gov/states/in/tax/income/deductions/in_unemployment_compensation_deduction.py

```diff
@@ -0,0 +1,31 @@
+from openfisca_us.model_api import *
+
+
+class in_unemployment_compensation_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN Unemployment compensation deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-2-10"
+    )
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states["in"].tax.income.deductions
+        unemployment_compensation_in_federal_AGI = tax_unit(
```

Comment:
And elsewhere; we always use lowercase.
```suggestion
        unemployment_compensation_in_federal_agi = tax_unit(
```